### DISCUSSION TITLE
父子应用都是umi构建的，开启qiankun插件，父应用的config里面配置了base之后 就挂载不到子应用了

### DISCUSSION THREAD

**Discussion by awishDay** [umijs/umi#7769] [Q&A]
禁用这个配置正常加载

**Comment by wswDotaIT** [comment]
与 @umijs/plugin-qiankun 2.3.0 之前版本的变化
主应用注册子应用时不再需要手动配置 base 和 mountElementId。

这类方式会导致很多关联问题，最典型的是如果我们需要将子应用挂载到某一个具体的子路由下时，常出现由于挂载点还未初始化或已被销毁导致的问题。

现在只需要在注册完子应用后，在期望的路由下指定需要挂载的子应用的 name 即可。

**Comment by cuijiudai** [comment]
我也遇到了
